subprojects/docs/src/docs/release/notes.md

```diff
@@ -34,22 +34,34 @@ For Java, Groovy, Kotlin and Android compatibility, see the [full compatibility
 <a name="cache"></a>
 ## Gradle's Dependency Cache can be relocated
 
-The Gradle Dependency cache under `$GRADLE_HOME/caches/modules-2` can now be relocated to another directory or host for dependencies cached by Gradle 6.1 and later. When moved to a new location or seeded into a host image, builds using the dependency cache will not need to access the network to download artifacts or metadata if the dependencies have already been downloaded. 
+It has become a common practice to run builds in a CI/CD pipeline with ephemeral agents, meaning single use environments.
+This practice is most often used to improve reliability and reduce potential interactions between builds.
+It however results in longer build times as nothing can be reused from build to build.
+
+A significant overhead caused by this practice with Gradle is the dependency downloads.
+Every build has a longer duration caused by the fetch of all dependencies.
+ 
+With Gradle 6.1, [the dependency cache](userguide/dependency_resolution.html#sec:dependency_cache) can be relocated, to another directory or host, for dependencies cached by Gradle 6.1 and later.
```

Comment:
```suggestion
The [dependency cache](userguide/dependency_resolution.html#sec:dependency_cache) of builds running with Gradle 6.1 can be relocated to another directory or host.
```
Avoid saying Gradle 6.1 twice. I think I know why you did it, but I think the reader won't get the detail from just this sentence. Important is imo the message that if you run all your builds with Gradle 6.1, then this works.